src/sdk/resources/policyActions.ts

```diff
@@ -0,0 +1,9 @@
+import { IHttpClient } from '../http';
+import { PagedResource } from './resource';
+import { PolicyAction } from '../models';
+
+export class PolicyActionsResource extends PagedResource<PolicyAction> {
+	constructor(httpClient: IHttpClient) {
+		super(httpClient, 'policy_action', 'policy_action', 'policy_actions', 'policy_actions', 'policies/actions');
+	}
+}
```

Comment:
Given that there is no direct link between Policies and Actions, the name `PolicyAction` and the route `policies/actions` are a little confusing.
Should we rename `PolicyActions` to something more accurate?
Let's PL tomorrow.

Reply:
In vnext maybe the Actions controller endpoints move into the TrackingGroupsController since these are technically group actions? Definitely a good PL topic for tomorrow though.

Reply:
Updated in new commit.